Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'whenever', '~> 0.9.4', require: false
 gem 'will_paginate'
 gem 'record_tag_helper', '~> 1.0'
 
+
```

Comment:
why the space?

Reply:
i believe i was trying to add a new gem for whatever reason and then decided it wasn't necessary, but forgot to remove the space. Sorry about that.